src/demo/index.php

```diff
@@ -68,6 +68,15 @@ function gtag() { dataLayer.push(arguments); }
                     <option>true</option>
                 </select>
 
+                <label for="date_format">Date Format</label>
+                <select class="param" id="date_format" name="date_format" placeholder="M j[, Y]">
+                    <option value="M j[, Y]">Aug 10, 2016</option>
+                    <option value="d F[, Y]">10 August, 2016</option>
+                    <option value="[Y,] M j">2016, Aug 10</option>
+                    <option value="j/n[/Y]">10/8/2016</option>
+                    <option value="n.j[.Y]">8.10.2016</option>
```

Comment:
Looks good :+1:

Not sure if we need a full month (eg. "August") option since that usually ends up making the lines too long to fit on the card. Also, the commas probably aren't necessary for dmy or ymd formats since the numbers aren't next to each other.

I suggest maybe the following options:

```suggestion
                    <option value="M j[, Y]">Aug 10, 2016</option>
                    <option value="j M[ Y]">10 Aug 2016</option>
                    <option value="[Y ]M j">2016 Aug 10</option>
                    <option value="j/n[/Y]">10/8/2016</option>
                    <option value="n/j[/Y]">8/10/2016</option>
                    <option value="[Y.]n.j">2016.8.10</option>
```